tonic_validate/metrics/answer_contains_pii_metric.py

```diff
@@ -0,0 +1,45 @@
+import os
+import requests
+from typing import List, Optional
+from tonic_validate.classes.llm_response import LLMResponse
+from tonic_validate.metrics.binary_metric import BinaryMetric
+from tonic_validate.services.openai_service import OpenAIService
+from tonic_textual.api import TonicTextual
```

Comment:
We should do the import inside the init function so we can catch the error if the user doesn't have textual installed and provide an error message